src/index.css

```
@@ -372,6 +372,422 @@ body {
+}
+
+.trace-card-raw,
+.trace-card-normalized {
+  margin: 0;
+  padding: 0.3rem 0.45rem;
+  border-radius: 6px;
+  background: var(--bg);
+  color: var(--ink);
+  font-size: 0.75rem;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)